examples/data_view_field_editor_example/public/app.tsx

```diff
@@ -136,7 +137,7 @@ const DataViewFieldEditorExample = ({ dataView, dataViewFieldEditor }: Props) =>
   );
 
   return (
-    <EuiProvider>
+    <EuiProvider theme={EuiThemeAmsterdam}>
```

Comment:
Looks like this change forces the dark mode, does not it?

<img width="1420" alt="Screenshot 2024-12-19 at 14 50 44" src="https://github.com/user-attachments/assets/988c426b-24f2-4bdf-a697-fb70aeaef52f" />

Reply:
@jughosta @stratoula 
I looked into it and as far as I can tell, this issue is not introduced by this PR.
The reason the examples are shown in Dark mode for you, is 
a) you have dark mode enables as OS settings
b) all examples use a separate `EuiProvider` which does not check on what color mode Kibana sets

EUI changed the `EuiProvider` a while back (version `95.12.0` ([release](https://github.com/elastic/eui/releases/tag/v95.12.0))) which enabled the `colorMode` to respect OS level settings _if_ no `colorMode` is set on the `EuiProvider` to override it. Previously the default `colorMode` was just `light`.

That means two things:
- this issue existed prior to this PR (and even on `main` I'm assuming) and would require all examples to either set a fixed `colorMode` or check on Kibanas colorMode to be in sync when updating color mode via the Kibana user settings.
- this issue even existed prio to the EUI change to respect OS level settings, just in the opposite direction: Examples would always load in light mode, but if Kibana is set to dark mode (user settings) then Kibana is dark but the examples still white

![Screenshot 2024-12-19 at 16 11 17](https://github.com/user-attachments/assets/4aec194a-9a27-4884-aaa0-5fd713832903)

If you agree, I'd say this is something that can/should be address as a separate change as it's not related to this PR or EUI specifically.

Reply:
Sure, thanks!

Reply:
Created https://github.com/elastic/kibana/issues/205033